docs/API/content-type/deleting-co.md

```diff
@@ -4,27 +4,195 @@ description: How to delete Content Objects in Flotiq
 
 # Deleting content objects
 
-Deleting of the object is done as the soft delete. All items will be still in the database, but not accessible by the API. It can be restored directly in the database, by setting `deleted_at` as `NULL` by the system administrator, if you need to bring back the deleted object, you can contact us on <a href="mailto:hello@flotiq.com">hello@flotiq.com</a>, please include your email, content type name and object id in the email.
-
-## Deleting single object
-
-Deleting is done by sending `DELETE` request to `https://api.flotiq.com/api/v1/content/{CTD name}/{object ID}`, where:
-
-* `CTD name` is the name of the content type definition
-* `object ID` is the ID of the object to remove
-
-For example: 
-
-```
-curl -X DELETE "https://api.flotiq.com/api/v1/content/blogposts/blogposts-1" -H "accept: application/json" -H "X-AUTH-TOKEN: YOUR_API_TOKEN"
-``` 
-
-For successful delete Flotiq will return `204 OK` response, for non-existing objects `404 Not Found` response, and for objects in relation within another object `400 Validation error` response.
+Deleting of the object is done as the soft delete. All items will be still in the database, but not accessible by the API.
```

Comment:
To be discussed how to tell the user that his data can be restored from backup etc.
This section should be moved to the bottom of this page (it's an edge case).

Reply:
This section will be moved to separate page